Userland/Libraries/LibJS/Bytecode/BasicBlock.cpp

```diff
@@ -10,19 +10,14 @@
 
 namespace JS::Bytecode {
 
-NonnullOwnPtr<BasicBlock> BasicBlock::create(DeprecatedString name, size_t size)
+NonnullOwnPtr<BasicBlock> BasicBlock::create(DeprecatedString name)
 {
-    return adopt_own(*new BasicBlock(move(name), max(size, static_cast<size_t>(4 * KiB))));
+    return adopt_own(*new BasicBlock(move(name)));
 }
 
-BasicBlock::BasicBlock(DeprecatedString name, size_t size)
+BasicBlock::BasicBlock(DeprecatedString name)
     : m_name(move(name))
 {
-    // FIXME: This is not the smartest solution ever. Find something cleverer!
-    // The main issue we're working around here is that we don't want pointers into the bytecode stream to become invalidated
```

Comment:
How does this PR ensure that this resize behavior does not happen?

Reply:
We should not need pointers into the bytecode stream. I am working on a set of follow-up changes that clarify this. :)

Reply:
https://github.com/SerenityOS/serenity/pull/21256